test/net/sourceforge/kolmafia/textui/RuntimeLibraryTest.java

```diff
@@ -86,4 +88,27 @@ void zapWandAvailable() {
       assertThat(output, containsString("name => marble wand"));
     }
   }
+
+  @Test
+  void floundryLocations() throws IOException {
```

Comment:
For example, this new test right here could use your html() function that catches IOException.